src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -496,6 +496,10 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                                     return $transitions;
                                                 }
 
+                                                if (!is_array($transitions)) {
```

Comment:
Shouldn't the same be done for places? What about adding some tests?

Reply:
Good idea! I updated the PR.